core/controlplane/config/templates/stack-template.json

```diff
@@ -207,7 +207,8 @@
                     { "Ref": "CloudWatchLogGroupARN" },
                     { "Fn::Join" : [ "", [{ "Ref": "CloudWatchLogGroupARN" }, ":log-stream:*"]] }
                   ]
-                },{{ end }}
+                },
+                {{ end }}
```

Comment:
Just added a new line after the json object since it is the standard in this file